tests/conftest.py

```diff
@@ -32,6 +33,11 @@ def teardown():
     return app_
 
 
+@pytest.fixture(scope='session')
+def initialized_db():
+    db.create_all()
```

Comment:
I expect you'll either want to extend this to add the static concepts et al, or add an alternative fixture to do so.
See https://github.com/uwcirg/truenth-portal/blob/develop/tests/__init__.py#L497 - many scenarios will require that static data to be resident in the db.

Reply:
Yup, we'll definitely have to add those setup steps as fixtures. it was an open question in my mind whether to create small fixtures that only test the bare minimum, or larger ones. I'm leaning towards creating minimal fixtures, but I'm worried they will be harder to maintain than fewer, larger fixtures.